public/app/features/logs/components/LogRowMenuCell.tsx

```diff
@@ -50,7 +54,7 @@ export const LogRowMenuCell = React.memo(
           (event.nativeEvent.ctrlKey || event.nativeEvent.metaKey || event.nativeEvent.shiftKey)
         ) {
           const win = window.open('about:blank');
-          const query = await getRowContextQuery(row);
+          const query = await getRowContextQuery(row, {}, true);
```

Comment:
```suggestion
          const query = await getRowContextQuery(row, undefined, true);
```

As we are not adding any options, this could be also passed as undefined.